src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -51,18 +55,34 @@ public virtual void Configure(IApplicationBuilder app)
         app.UseDevelopmentIdentityProviderIfConfigured();
     }
 
-    /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
-    /// </summary>
-    private void AddApplicationInsightsTelemetry(IServiceCollection services)
+    private void AddTelemetry(IServiceCollection services)
     {
         string instrumentationKey = Configuration["ApplicationInsights:InstrumentationKey"];
 
         if (!string.IsNullOrWhiteSpace(instrumentationKey))
         {
             var connectionString = $"InstrumentationKey={instrumentationKey}";
-
-            services.AddApplicationInsightsTelemetry(aiOptions => aiOptions.ConnectionString = connectionString);
+            AddOpenTelemetry(services, connectionString);
+            AddApplicationInsightsTelemetry(services, connectionString);
         }
     }
+
+    /// <summary>
+    /// Adds Open telemetry exporter for Azure monitor.
+    /// </summary>
+    private static void AddOpenTelemetry(IServiceCollection services, string connectionString)
```

Comment:
Maybe call this `AddOpenTelemetryMetrics` to be specific?

Reply:
Done